packages/manager/apps/dedicated/client/app/components/ip/ip-ip-organisation.service.js

```diff
@@ -97,7 +97,7 @@ export default /* @ngInject */ function IpOrganisationService(
       if (task) {
         self.poll({
           taskId: task.taskId,
-          ipBlock: encodeURIComponent(params.ipBlock),
+          ipBlock: params.ipBlock,
```

Comment:
**question:** Are you sure to remove `encodeURIComponent`?

Reply:
Yes Anoop. Its working fine without encoding. If we add it api is throwing error.